x/leverage/gov_handler_test.go

```diff
@@ -89,6 +67,6 @@ func TestUpdateRegistryProposalHandler(t *testing.T) {
 
 		token, err := k.GetTokenSettings(ctx, "uosmo")
 		require.NoError(t, err)
-		require.Equal(t, "0.300000000000000000", token.ReserveFactor.String())
+		require.Equal(t, "0.200000000000000000", token.ReserveFactor.String())
```

Comment:
This test is for a proposal modifying the reserve factor of an existing token from 0.2 to 0.3. Making the expected value to 0.2 should not be done, because a failure to update the registry would be misinterpreted as a success.

Reply:
thanks for spotting

Reply:
updated